.github/workflows/publish-github.yml

```diff
@@ -0,0 +1,54 @@
+name: Publish
+
+on:
+  workflow_call:
+    inputs:
+      target:
+        type: string
+        required: true
+      arch:
+        type: string
+        required: true
+      dist:
+        type: string
+        required: true
+      package-type:
+        type: string
+        required: true
+      environment:
+        type: string
+        description: The GitHub environment to use for the build.
+        required: true
+      release-name:
+        type: string
+        description: The GitHub release name to update.
+        required: true
```

Comment:
This is TBD for future, correct? Here we'll have something line this: [https://github.com/Azure/azure-osconfig/releases/tag/test_policy_package](https://github.com/Azure/azure-osconfig/releases/tag/test_policy_package)?

Reply:
Thats correct, its not being used yet, this will publish to a particular release name (existing release)